src/librustdoc/html/static/rustdoc.css

```diff
@@ -768,16 +768,29 @@ a {
 	display: block;
 }
 
-.search-results a {
+.search-results > a {
 	/* A little margin ensures the browser's outlining of focused links has room to display. */
 	margin-left: 2px;
 	margin-right: 2px;
 	display: block;
 }
 
-.result-name {
+.search-results > a > div {
+	display: flex;
+}
+
+.search-results > a > div > div {
```

Comment:
When we have a series of direct-parent selectors like that, we're creating a brittle dependency on a specific structure of the DOM. It becomes hard to make a change to the code that generates search results and figure out which CSS selectors need to change.

Instead, we should add classes, or use the classes that are already here - `result-name` and `desc` (though note `desc` is on a span rather than a div and might not be exactly what you're looking for).

Reply:
Good point.